test/appium/tests/atomic/chats/test_public.py

```diff
@@ -435,6 +435,38 @@ def test_different_status_in_timeline(self):
 
         self.errors.verify_no_errors()
 
+    @marks.testrail_id(700727)
+    @marks.medium
+    def test_gap_in_public_chat(self):
+        self.create_drivers(2)
+        device_1, device_2 = SignInView(self.drivers[0]), SignInView(self.drivers[1])
+        home_1, home_2 = device_1.create_user(), device_2.create_user()
+        profile_1, profile_2 = home_1.profile_button.click(), home_2.profile_button.click()
+        profile_1.sync_settings_button.click()
+        profile_2.sync_settings_button.click()
+        profile_1.sync_history_for_button.click()
+        profile_2.sync_history_for_button.click()
+        profile_1.element_by_translation_id("two-minutes").click()
+        profile_2.element_by_translation_id("two-minutes").click()
+        profile_1.home_button.click()
+        profile_2.home_button.click()
+        chat_1 = home_1.join_public_chat("pavlotest")
+        chat_2 = home_2.join_public_chat("pavlotest")
+        chat_1.send_message("HI")
+        # chat_1.home_button.click()
+        device_1.toggle_airplane_mode()
+        chat_2.send_message("testing gap")
+        sleep(180)
+        device_1.toggle_airplane_mode()
+        sleep(120)
+        chat_1.element_by_translation_id("fetch-messages").click()
+        chat_1.chat_element_by_text("testing gap")
```

Comment:
There is no actual check.
Should be is_element_displayed in particular case

Reply:
thanx, @churik , added is_element_displayed() method for check